developers/weaviate/more-resources/migration/weaviate-1-30.md

```diff
@@ -0,0 +1,181 @@
+---
+title: 1.30 (BlockMax WAND)
+sidebar_position: 9
+image: og/docs/more-resources.jpg
+# tags: ['migration']
+---
+
+# Weaviate 1.30 migration guide
+
+This guide covers how to migrate existing data to use the BlockMax WAND algorithm for the inverted index.
+
+## Assumptions & requirements
+
+This migration guide assumes that you have:
+
+- A Weaviate version before `v1.30` that doesn't use the BlockMax WAND algorithm
+- Async replication enabled and a properly initialized hashtree (verified by logs showing either `"msg":"hashtree successfully initialized"` or `hashbeat iteration successfully completed: no differences were found`)
+
+## Migration overview
+
+All BM25 and hybrid searches will use BlockMax WAND algorithm for searches, potentially improving search performance.
+However, any existing collections will continue to use the default (pre-BlockMax WAND) disk structure and search algorithm. To take advantage of BlockMax WAND, you must migrate your existing collections.
+
+This migration process involves:
+1. Converting the `property_<property_name>_searchable` from `mapcollection` to the inverted `blockmax` format
+2. Swapping the original buckets to use the new format
+3. Cleaning up old data after successful migration
+
+:::note
+This guide is subject to changes as the process is refined. The latest image tag to use for migration is:  
+`semitechnologies/weaviate:1.29.1-ea997c7`
+:::
+
+### Step 1: Start the migration
+
+1. Restart your Weaviate instance with the following configuration:
+
+```
+USE_INVERTED_SEARCHABLE=true
+USE_BLOCKMAX_WAND=true
+REINDEX_MAP_TO_BLOCKMAX_AT_STARTUP=weaviate
+REINDEX_MAP_TO_BLOCKMAX_SWAP_BUCKETS=weaviate
+MAINTENANCE_NODES=weaviate
+```
+
+:::note
+If your node has a different name than "weaviate", replace it with your actual node name in the configuration above.
+:::
+
+2. Monitor the migration progress by checking:
+   ```
+   http://localhost:6060/debug/index/rebuild/inverted/status?collection=Products_v2
+   ```
+   
+   Replace "Products_v2" with your actual collection name if different.
+
+   The migration process may take some time to start (up to an hour, depending on shard vector initialization time).
```

Comment:
Does it take this long to _start_? 😮 Or is that to complete

Reply:
This is specific for the LTK dataset, removed